### ISSUE TITLE
btrfs: extra double quotes

### ISSUE THREAD

**Issue by ghost** [mrlesmithjr/ansible-manage-lvm#8]
The extra double quotes at the end of the line triggers an error.
```
ERROR! Syntax Error while loading YAML.

The error appears to have been in '/xxx/roles/ansible-manage-lvm/tasks/manage_lvm.yml': line 66, column 57, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

- name: manage_lvm | resizing btrfs
  shell: "btrfs filesystem resize max {{ item.1.mntp }}""
                                                        ^ here
```

**Comment by mrlesmithjr** [comment]
@olo-dw Fixed. Thanks for pointing this out.

**Comment by ghost** [comment]
Fixed by commit 0d255cb5ce9e6016a503203c2ed39a70fb0ccd9c.
